changelog.txt

```diff
@@ -1,123 +1,314 @@
-0.4.1
-
+=========
+Changelog
+=========
+
+The **garnett** package follows `semantic versioning <https://semver.org/>`_.
+
+Version 0.5
+===========
+
+Highlights
+----------
+	- Remove acceleration as supported property.
+	- Remove the read_gsd_shape_data flag from GSD reader.
+	- Raise AttributeError if accessing an invalid property.
+	- Add GETAR file reader/writer.
+	- Add support for various particle properties
+	- Extend reader/writer shape support
+	- Add automatic format detection to reader/writer
+	
+
+[0.5.0] -- 2019-0x-xx
+---------------------
+Added
++++++
+
+	- Add GETAR file reader/writer.
+	- Rowan is now a dependency.
+	- Add shape_dict representation to Shape classes.
+	- Add support for particle properties: 
```

Comment:
It appears there is a lot of extra whitespace at the ends of lines like this one. (I recommend that everyone should configure their code editor to automatically remove whitespace at the ends of lines, so this never happens by accident.)

Reply:
I only saw one extra whitespace (2ad7399), not sure if I missed any others

Reply:
@klywang I think @bdice also refers to the trailing spaces at the end of lines

Reply:
673ca89 Trailing spaces at ends of line also removed